src/tests/Stub.list

```diff
@@ -12,6 +12,7 @@ STUB_SOURCE= tests/STUB.h \
 	tests/stub_cache_cf.cc \
```

Comment:
@kinkie, it looks like this generated Makefile include file does not go through the formatting script that you added in 2a06115. IMO, we should change the file extension to `.am` (which we already use for other Makefile include files) so that it is properly formatted. If you volunteer to fix this, please also check that this file is generated _before_ Makefiles are formatted. Thank you!

Reply:
This is being handled in other PR.